### DISCUSSION TITLE
SCSS in top languages card

### DISCUSSION THREAD

**Discussion by mathisneunzig** [anuraghazra/github-readme-stats#1805] [Q&A] [answered]
Hi!

I have some CSS in my repos and it shows up in my top 10 languages card. There is some CSS, but I normally use almost only SCSS/SASS right now - and there is waaay more SCSS then CSS. There is also a repo where scss is the top language - but it is not showing up in my card. Why does it show CSS but not SCSS?

**Comment by rickstaa** [comment, marked as answer]
@DrBackmischung Thanks for your answer. I checked your account and it looks like the data we get returned shows less CSS than SCSS. You can try this out by running the following code in the GraphQL explorer:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

With variables: 

```graphql
{"login": "DrBackmischung"}
```

It gives me 2120 `scss` bytes and 11567 `CSS` bytes. This will result in the following card:

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=DrBackmischung&langs_count=10)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=DrBackmischung&langs_count=10)](https://github.com/anuraghazra/github-readme-stats)

That being said, there are multiple reasons for the card not showing as expected:
- In your reasoning, you include data from forked repositories. As explained in #1255 and #1803 we can not include forks.
- In your reasoning, you include data from private repositories. As explained in https://github.com/anuraghazra/github-readme-stats/discussions/1770#how-to-count-private-stats the public Vercel instance can not access data from private repositories. You, therefore, must deploy your own Vercel instance and add a Personal Access token if you want to include private data. See https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance for more information.
- The language counting algorithm currently is very brute force. Due to the current GraphQL implementation, we can only get the language results of repositories and not the language results of individual users (for more information, see https://github.com/anuraghazra/github-readme-stats/pull/1122#issuecomment-1152066225).
- Further, we currently do not include data from repositories you contributed to but only the data of repositories if you are the owner. The reason we do this is related to the point above. For more information, see the tread in #1122.

**Reply by rickstaa** [reply]
@DrBackmischung Does that solve your problem?
